go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/PaddleHQ/paddle-go-sdk
 go 1.21.0
 
 require (
-	github.com/ggicci/httpin v0.18.1
+	github.com/ggicci/httpin v0.18.2-0.20240520043217-eed0621e7f51
```

Comment:
we don't currently have a tagged version available. 
I am hoping this will be added soon - and will update the PR once this is ready.